.ci_support/test_environment.yml

```diff
@@ -0,0 +1,15 @@
+channels:
+  - conda-forge
```

Comment:
Please use the same naming used in the user pyiron modules to be consistent - https://github.com/pyiron/pyiron/blob/master/.ci_support/environment-notebooks.yml

Reply:
Do you mean the name of the file should be changed to `environment-test.yml`?

Reply:
I would name it `environment-notebooks.yml`